.github/workflows/rust.yml

```diff
@@ -11,8 +11,11 @@ env:
 
 jobs:
   build:
-
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
```suggestion
        # windows-latest does not work because of https://github.com/oli-obk/ui_test/issues/147
        os: [ubuntu-latest, macos-latest]
```